students/ArchWu/Session3/slicing_lab.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+def exchange_first_last(seq):
+    a_new_sequence = seq[-1:] + seq[1:-1] + seq[:1]
+    return a_new_sequence
+
+def exchange_first_last_remove_else(seq):
+    a_new_sequence = seq[-1:] + seq[:1]
+    return a_new_sequence
+
+def remove_first4_last4(seq):
+    a_new_sequence = seq[4:-4]
+    return a_new_sequence
+
+def reverse(seq):
+    a_new_sequence = seq[::-1]
+    return a_new_sequence
+
+def thirds(seq):
+    a_new_sequence = seq[2 * len(seq)//3:] + seq[:len(seq)//3] + seq[len(seq)//3:2*len(seq)//3]
```

Comment:
probably worth assigning a variable to `len(seq)//3` -- rather than re-computing it four times.

but otherwise, exactly right.

Reply:
Very reasonable, will make changes